speech/grpc/pom.xml

```diff
@@ -112,6 +112,11 @@ limitations under the License.
   <!-- // [START dependency] -->
   <dependencies>
     <dependency>
+      <groupId>com.google.cloud</groupId>
```

Comment:
Actually after switching to use the URI parameter, https://cloud.google.com/speech/reference/rest/v1/speech/recognize#audiorequest you should not need this dependency.